src/components/common/popup-model.tsx

```
@@ -98,10 +98,49 @@ const PopUpModal = () => {
+                    fontWeight: 'bold',
+                    color: '#000000',
+                    lineHeight: '1',
+                    opacity: '1',
+                    zIndex: '9999',
+                    WebkitTextFillColor: '#000000',
+                    textShadow: 'none'
+                  }}
+                >
+                  <span style={{ color: '#000000', WebkitTextFillColor: '#000000' }}>×</span>
+                </button>
+                <style dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)